### ISSUE TITLE
page request: document stderr to stdout

### ISSUE THREAD

**Issue by corneliusroemer** [tldr-pages/tldr#9460]
### Command description

I keep forgetting how to redirect stderr to stdout - would it be possible to document this in tldr in some way?

Not sure what the best way would be since this isn't quite a normal command but a bash inbuilt.

Any clues?

`2>&1`

### OS

- [ ] Android
- [X] Linux
- [X] macOS
- [ ] sunOS
- [ ] Windows

**Comment by navarroaxel** [comment]
@waldyrious wdyt?

**Comment by corneliusroemer** [comment]
Maybe we could create a command `stderr`? Has anyone ever added bash intrinsics to `tldr`?

**Comment by CleanMachine1** [comment]
We have a `[[` page. Can this be even called a command more than just a syntax feature of bash.

I think we could add this as `tldr 2>&1` and maybe think about documenting such features as `&&` as well.

**Comment by EmilySeville7cfg** [comment]
> I keep forgetting how to redirect stderr to stdout - would it be possible to document this in tldr in some way?

It will be possible only when it will be permitted by our client specification. Now it literally states the following:

> tldr-pages consists of multiple pages - each of which describes a specific **command**.

By all means, redirection can be shown in some tldr pages but it's unclear what page to use to find i/o redirection syntax. So my point is: update client specification if users need show syntax features to be shown separately from usual tldr pages.

**Comment by Managor** [comment]
We now have the pages that document this: https://github.com/tldr-pages/tldr/pull/13846
